packages/editor/src/components/post-publish-panel/style.scss

```diff
@@ -4,6 +4,9 @@
 }
 
 .editor-post-publish-panel__content {
+	// Ensure the post-publish panel accounts for the header and footer height.
+	min-height: calc(100% - #{ $header-height + 60px });
```

Comment:
Is the footer height not in a variable? If so we should make one or this'll get out-of-date and regress when changed.

Reply:
If I'm not wrong, the publish panel footer height is not set via CSS but just determined by its content. Foro example, longer translations might increase its actual height which would be taller than 60px:

<img width="324" alt="screen shot 2018-10-17 at 12 09 50" src="https://user-images.githubusercontent.com/1682452/47079466-e6951700-d205-11e8-9619-60040045c9b7.png">

Reply:
What Andrea said. To avoid scrolling when the translations take two rows and the accordions are closed, we can make this 84px. It would work in mobile as well.

```suggestion
	min-height: calc(100% - #{ $header-height + 84px });
```